src/main/java/Duke.java

```diff
@@ -1,10 +1,179 @@
+                System.out.println("    Please specify task: ");
+                inputString = getInputString();
+                if (inputString.equals("")) {
+                    System.out.println("☹ OOPS!!! The description of a todo cannot be empty.");
+                    break;
+                } else {
+                    tasks.add(new Todo(inputString));
+                    tasks.get(counter).print();
+                    counter++;
+                    System.out.println("    Now you have " + counter + " tasks in your list!");
+                    break;
+                }
+
+            case "deadline":
+                System.out.println("    _________________________________________");
+                System.out.println("    Please specify task: ");
+                inputString = getInputString();
+                inputString += '\0';
+                int deadlinePosition = inputString.indexOf("/");
+                int endPosition = inputString.indexOf("\0");
+                String taskName = inputString.substring(0, deadlinePosition);
+                String deadline = inputString.substring(deadlinePosition + 1, endPosition);
+                tasks.add(new Deadline(taskName, deadline));
+                tasks.get(counter).print();
+                counter++;
+                break;
+
+            case "event":
+                System.out.println("    _________________________________________");
+                System.out.println("    Please specify task: ");
+                inputString = getInputString();
+                inputString += '\0';
+                int deadlineStartPosition = inputString.indexOf("/");
+                int deadlineEndPosition = inputString.indexOf("|");
+                endPosition = inputString.indexOf("\0");
+                taskName = inputString.substring(0, deadlineStartPosition);
+                String deadlineStart = inputString.substring(deadlineStartPosition + 1, deadlineEndPosition);
+                String deadlineEnd = inputString.substring(deadlineEndPosition + 1, endPosition);
+                tasks.add(new Event(taskName, deadlineStart, deadlineEnd));
+                tasks.get(counter).print();
+                counter++;
+                break;
+
+            case "delete":
+                System.out.println("    Please specify task number: ");
+                taskNumber = getTaskNumber() - 1;
+                System.out.println("    _________________________________________");
+                tasks.get(taskNumber).printInList();
+                System.out.println("    _________________________________________");
+                System.out.println("    ");
+                tasks.remove(tasks.get(taskNumber));
+                counter--;
+                break;
+
+            default:
+                System.out.println("☹ OOPS!!! I'm sorry, but I don't know what that means :-(");
+            break;
+        }
+    }
```

Comment:
the main method seems to be doing everything. 
please refactor the code to extract methods for the operations done in main